src/agent/debugger/src/stack.rs

```diff
@@ -86,13 +85,25 @@ pub enum DebugStackFrame {
     Frame {
         function: String,
         location: DebugFunctionLocation,
+        symbol: Option<String>,
+        module_name: Option<String>,
     },
     CorruptFrame,
 }
 
 impl DebugStackFrame {
-    pub fn new(function: String, location: DebugFunctionLocation) -> DebugStackFrame {
-        DebugStackFrame::Frame { function, location }
+    pub fn new(
+        function: String,
```

Comment:
This is a little confusing now, maybe it'd be better to remove the `function` field.

That said, I'm not sure it's a good idea to allow frames with no module or symbol as you have no position to render.

Obviously some locations won't have a module (e.g. JIT compiled code) and some locations won't have symbols (again, JIT compiled code).

I do think it's useful to call out an unknown module. The current rendering uses the VA though, and that's less useful for hashing as it is likely not stable. Better would be to convert to some sort of RVA, perhaps based on the memory region's base address (`win_util::memory::get_memory_info` can return the base address of the region for a given address.)

Reply:
Before this PR, it looks like we set `function` to the _module name_ when we can't find the symbol info: https://github.com/microsoft/onefuzz/blob/main/src/agent/debugger/src/stack.rs#L209-L212

So if we're going to change the `DebugStackFrame::Frame` variant, it might be a good opportunity to clean that up, too. We should always have a module (even if just `UNKNOWN_MODULE`) and relative offset, and that's part of what makes the current definition even harder to follow, I think. The optionality is sort of backwards.

Reply:
> Better would be to convert to some sort of RVA, perhaps based [...]

@lzybkr, this is exactly what you impl'd [here](https://github.com/microsoft/onefuzz/blob/main/src/agent/debugger/src/stack.rs#L225-L227), right?

Reply:
Yep, forgot about that, though maybe that's why I added `get_memory_info` in the first place.